tests/rados/test_scrub_parameters.py

```diff
@@ -0,0 +1,252 @@
+                scheduled_time = verify_scheduled_time(
+                    mon_object=mon_obj,
+                    prev_dump_scrub=dump_scrub_before_scrub,
+                    pg_id=pg_id,
+                    pg_current_sched_time=pg_current_sched_time,
+                )
+                if not scheduled_time:
+                    log.error("The scrub time is greater than upper bound time")
+                    return 1
+                log.info(f"===Scenario-2:Completed by {val}bing the osd")
+                log.info(f"=====Verification completed by {val}bing the OSD====")
+                # deleted pool.This is due to execution time is taking more time until waiting for the cluster to clean
+                # state
+                method_should_succeed(rados_object.delete_pool, pool_name)
+                time.sleep(10)
+                rados_object.log_cluster_health()
+    except Exception as e:
+        log.info(e)
+        log.info(traceback.format_exc())
+        return 1
+    finally:
+        log.info("Execution of finally block")
+        method_should_succeed(rados_object.delete_pool, pool_name)
+        rados_object.log_cluster_health()
+    return 0
+
+
+def create_pool_get_osd(rados_object, pool_type, pool_config):
+    """
+    Method is used -
+        1. Create replicated pool
+        2. Push the data into pool
+        3. From the active OSD list return a random OSD
+    Args:
+        rados_object: Rados object
+        pool_name: pool name
+    Returns:
+        Return a random OSD ID from an active OSD list
+    """
+    pool_name = pool_config["pool_name"]
+    if pool_type == "replicated_pool":
+        if not rados_object.create_pool(**pool_config):
+            log.error("Failed to create the replicated Pool")
+            return 1
+    else:
+        if not rados_object.create_erasure_pool(name=pool_name, **pool_config):
+            log.error("Failed to create the EC Pool")
+            return 1
+
+    if not rados_object.bench_write(
+        pool_name=pool_name, byte_size="5M", max_objs=100, verify_stats=True
+    ):
+        log.error("Failed to write objects into Pool-1, with compression enabled")
+        return 1
+    res = wait_for_clean_pg_sets(rados_object, test_pool=pool_name)
+    if not res:
+        log.error("PG's in cluster are not active + Clean ")
+        return 1
+    osd_list = rados_object.get_active_osd_list()
+    osd_id = random.choice(osd_list)
+    return osd_id
+
+
+def chk_forced_flag(scrub_object, osd_id):
+    """
+    Method is used to check the forced flag status
+    Args:
+        rados_object: Rados object
+        osd_id: OSD ID number
+    Returns:
+        True -> If forced flag is true
+        False -> If forced flag is false
+    """
+    pg_id = ""
+    time_execution = datetime.datetime.now() + datetime.timedelta(minutes=20)
+    found = False
+    while datetime.datetime.now() < time_execution:
+        out_put = scrub_object.get_dump_scrubs(osd_id)
```

Comment:
Please add a debug statement printing `out_put` here. it might be useful during debugging sessions.

Reply:
Debug statement is added.